cypress/integration/dates_spec.js

```diff
@@ -0,0 +1,6 @@
+describe('Timeline', function() {
+  it('can display dates', function() {
+    cy.visit('/posts');
+    cy.get('.posts').should('contain', 'British Summer Time');
```

Comment:
Should we add a step where we add a post? When we are able to have the DB clear before each integration test then this test might fail?

Reply:
sure, can do

Reply:
done :)